src/openvslam/tracking_module.cc

```diff
@@ -333,15 +360,32 @@ bool tracking_module::track_current_frame() {
         if (!succeeded) {
             succeeded = frame_tracker_.robust_match_based_track(curr_frm_, last_frm_, ref_keyfrm_);
         }
+    } else if (update_pose_is_requested()) {
+        // Force relocalization by pose
+        curr_frm_.set_cam_pose(request_pose_);
+
+        curr_frm_.compute_bow();
+        const auto candidates = map_db_->get_close_keyframes(request_pose_, update_pose_keyframes_);
+        if (candidates.size() != 0) {
+            succeeded = relocalizer_.reloc_by_candidates(curr_frm_, candidates);
+        }
+
+        if (succeeded) {
+            last_reloc_frm_id_ = curr_frm_.id_;
+            tracking_state_ = tracker_state_t::Tracking;
+        } else {
+            tracking_state_ = tracker_state_t::Lost;
```

Comment:
As with the existing relocalization, there is no need to set it here as it will be set later.
https://github.com/OpenVSLAM-Community/openvslam/blob/337d89edc53d7135c73361507795dd25b4d5e62a/src/openvslam/tracking_module.cc#L286

Reply:
Fixed [here](https://github.com/OpenVSLAM-Community/openvslam/pull/79/commits/ed3ce6c2d7549ee101db12e41b74f92ac1a3411f#diff-9d7c6e92591d28cb8c87b3973729fddb3abd564e9feae8861851fd48dd31b2caR370-R372).